examples/terraform/vmware-cloud-director/main.tf

```diff
@@ -194,119 +194,8 @@ resource "vcd_nsxv_firewall_rule" "rule_internet" {
 resource "vcd_nsxv_snat" "rule_internet" {
   edge_gateway = data.vcd_edgegateway.edge_gateway.name
   network_type = "ext"
-  network_name = local.external_network
+  network_name = local.external_network_name
 
-  original_address   = var.network_subnet
-  translated_address = local.public_ip
-}
-
-# Create the firewall rule to allow SSH from the Internet
-resource "vcd_nsxv_firewall_rule" "rule_ssh_bastion" {
-  edge_gateway = data.vcd_edgegateway.edge_gateway.name
-  name         = "${var.cluster_name}-firewall-rule-ssh"
-
-  action = "accept"
-
-  source {
-    ip_addresses = ["any"]
-  }
-
-  destination {
-    ip_addresses = [local.public_ip]
-  }
-
-  service {
-    protocol = "tcp"
-    port     = 22
-  }
-}
-
-# Create DNAT rule to allow SSH from the Internet
-resource "vcd_nsxv_dnat" "rule_ssh_bastion" {
-  edge_gateway = data.vcd_edgegateway.edge_gateway.name
-  network_type = "ext"
-  network_name = local.external_network
-
-  original_address = local.public_ip
-  original_port    = 22
-
-  translated_address = vcd_vapp_vm.bastion.network[0].ip
-  translated_port    = 22
-  protocol           = "tcp"
-}
-
-#################################### Loadbalancer ####################################
```

Comment:
Why we don't have an LB any longer?

Reply:
Their load-balancer service doesn't work well with NSX-V. Also, we'd preferably like to keep the nodes, kube-apiserver private just like we do for vSphere.